real_time/surface_src/java_src/Alice/src/test/java/TestWayPointFollowerPlanner.java

```diff
@@ -51,4 +56,12 @@ public void test() {
 		
 		}
 
+	@Test
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382738414/issues/source?bid=2938665&fileBranchId=3226464#l60)